apps/frontend/src/pages/common-layout/styles.module.css

```diff
@@ -0,0 +1,19 @@
+.container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-primary);
+}
+
+.container-sidebar-content {
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+}
+
+.content-container {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+}
```

Comment:
As I mentioned above, maybe it would be better to add `padding: 32px 80px` here, instead of adding its for every component like `AccessManagement`